CondFormats/HcalObjects/src/HcalInterpolatedTableFunctor.cc

```diff
@@ -0,0 +1,101 @@
+#include <algorithm>
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "CondFormats/HcalObjects/interface/HcalInterpolatedTableFunctor.h"
+
+inline static double interpolateStep(
```

Comment:
Hm... I had an impression that static functions
always have internal linkage. Where do I read that
it becomes visible if one compiles with -O0 ?

On 08/04/2016 04:42 PM, Slava Krutelyov wrote:
>
> In CondFormats/HcalObjects/src/HcalInterpolatedTableFunctor.cc <https://github.com/cms-sw/cmssw/pull/15324#discussion_r73608291>:
>
> > @@ -0,0 +1,101 @@
> > +#include <algorithm>
> > +#include "FWCore/Utilities/interface/Exception.h"
> > +
> > +#include "CondFormats/HcalObjects/interface/HcalInterpolatedTableFunctor.h"
> > +
> > +inline static double interpolateStep(
>
> put this in an anonymous namespace or make less randomly reproducible name?
> chances of collision are probably low though (also, one will need to compile with -O0 to make it a visible object)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/cms-sw/cmssw/pull/15324/files/f2734c7b29b226480f1efc887de88c8ae73f121f#r73608291>, or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFS4-bFAdXXWJEhT7KvCbj2vJxTf36pnks5qclzBgaJpZM4JXio9>.
>